src/main/haskell/language-kore/test/parser/Data/Kore/Step/BaseStepTest.hs

```diff
@@ -0,0 +1,777 @@
+                        [ StepProofVariableRenamings
+                            [ variableRenaming
+                                (a1 PatternSort) (var_0 PatternSort)
+                            , variableRenaming
+                                (x1 PatternSort) (var_1 PatternSort)
+                            ]
+                        , StepProofUnification
+                            ( proposition_5_24_3
+                                [ functionalVariable (a1 PatternSort)
+                                ]
+                                (var_1 PatternSort)
+                                (a1 PatternSort)
+                            )
+                        ]
+                    )
+                )
+                (runStep
+                    mockMetadataTools
+                    StepperConfiguration
+                        { stepperConfigurationPattern =
+                            asPureMetaPattern (a1 PatternSort)
+                        , stepperConfigurationConditionSort = asAst SortSort
+                        , stepperConfigurationCondition = top SortSort
+                        }
+                    AxiomPattern
+                        { axiomPatternLeft =
+                            asPureMetaPattern (x1 PatternSort)
+                        , axiomPatternRight =
+                            asPureMetaPattern
+                                (metaExists
+                                    PatternSort
+                                    (a1 PatternSort)
+                                    (x1 PatternSort)
+                                )
+                        }
+                )
+            )
+        -- sigma(x, x) -> x   vs   sigma(g(a), f(b))
+        , testCase "Symbol clashes."
+            (assertEqualWithExplanation ""
+                (Left $ StepErrorUnification $ ConstructorClash gSymbol fSymbol)
+                (runStep
+                    mockMetadataTools
+                    StepperConfiguration
+                        { stepperConfigurationPattern =
+                            asPureMetaPattern
+                                ( metaSigma
+                                    (metaG (a1 PatternSort))
+                                    (metaF (b1 PatternSort))
+                                )
+                        , stepperConfigurationConditionSort = asAst SortSort
+                        , stepperConfigurationCondition = top SortSort
+                        }
+                    AxiomPattern
+                        { axiomPatternLeft =
+                            asPureMetaPattern
+                                (metaSigma (x1 PatternSort) (x1 PatternSort))
+                        , axiomPatternRight =
+                            asPureMetaPattern (x1 PatternSort)
+                        }
+                )
+            )
+        -- sigma(sigma(x, x), sigma(y, y)) -> sigma(x, y)
+        -- vs
+        -- sigma(sigma(a, f(b)), sigma(b, a))
```

Comment:
maybe you should consider adding the expected result of the one step, too.  I know it should be obvious, but still...

Reply:
Done.